README.md

```diff
@@ -52,23 +61,40 @@ Welcome to the **CarLens** project. It's an internal Netguru's application, whic
 	carthage bootstrap --platform iOS --cache-builds
 	```
 
-4. Download `.env` file from project's 1password vault and paste it into the root project's directory.
+4. Rename `.env.sample` to `.env`. If you're Netguru employee update it with values from project's 1password vault.
```

Comment:
I'd remove the info for Netguru employees. I'm not sure we want to give everybody a hint of what password manager we use. 😊

Reply:
True, that was something I was doubting too. The repository is public now and there were also links to JIRA, Bitrise, Hockey App, etc. :D But I also think it's not needed, gonna remove it, thanks!